lib/chrome/find-progids.js

```diff
@@ -29,7 +29,11 @@ function queryHive (self, hive, done) {
         if (err) return next(err.notFound ? null : err)
 
         const metadata = {
-          channel: id === 'ChromeHTML' ? 'stable' : 'canary'
+          channel: id.startsWith('ChromeSSHTM.') ? 'canary' : {
+            'ChromeHTML': null, // In some cases this can be ambiguous
+            'ChromeBHTML': 'beta',
+            'ChromeDHTML': 'dev'
+          }[id]
```

Comment:
Can you move this logic to a function? Along the lines of:

```js
function channel (id) {
  if (id.startsWith('ChromeSSHTM.')) return 'canary'
  if (id === 'ChromeBHTML') return 'beta'
  // etc
}
```

Reply:
Done :+1: